README.md

```diff
@@ -36,11 +36,43 @@ Follow the latest libp2p implementations at
 - [cpp-libp2p](//github.com/soramitsu/libp2p) in C++
 - [swift-libp2p](//github.com/swift-libp2p/swift-libp2p) in Swift
 - [nim-libp2p](//github.com/status-im/nim-libp2p) in Nim
-- [jvm-libp2p](//github.com/libp2p/jvm-libp2p) in Kotlin
+- [jvm-libp2p](//github.com/libp2p/jvm-libp2p) in Kotlin and Java
 - [erlang-libp2p](//github.com/helium/erlang-libp2p) in Erlang
 - [zig-libp2p](//github.com/marcopolo/zig-libp2p) in Zig
 - [dotnet-libp2p](//github.com/NethermindEth/dotnet-libp2p) in .NET
 
-## Community Discussion
+## Community 
 
-Please visit our discussion forums at [**discuss.libp2p.io**](https://discuss.libp2p.io) to get help, ask questions about the past, present, and future of libp2p, and more.
+### Announcements
+
+For all **lib**p2p community announcements look for posts under the [**"News" topic in the community-wide forums**](https://discuss.libp2p.io/c/news/8) or tagged as an announcement in the technical discussion forums listed below or also announced in the [**#libp2p-implementers channel on Filecoin Slack**](https://filecoinproject.slack.com/archives/C03K82MU486). 
+
+### Blog
+
+The **lib**p2p community blog can be read at [**blog.libp2p.io**](https://blog.libp2p.io). If you would like to write a blog post about your cool project that uses **lib**p2p, you can either post a message in [**this project's discussion forum**](https://github.com/libp2p/libp2p/discussions) or you may post a message in our community-wide discussion forums at [**discuss.libp2p.io**](https://discuss.libp2p.io). If you tag either @p-shahi or @dhuseby in your post we'll get back to you ASAP about your post.
```

Comment:
In thatcase, rather than posting a discussion in https://github.com/libp2p/libp2p/discussions, should have the discussion in https://github.com/libp2p/blog?

I'm partly bringing this up because there is then some confusion on when one uses:
https://github.com/libp2p/libp2p/discussions vs. https://discuss.libp2p.io/